cmd/director.go

```diff
@@ -70,6 +70,13 @@ func getDirectorEndpoint() (string, error) {
 	return directorEndpointURL.String(), nil
 }
 
+func initDirector() error {
+	err := config.InitServer()
```

Comment:
Why do we want to pull this into a separate function if we don't have anything else to add?

Reply:
It matches what we do with the origin [see here](https://github.com/PelicanPlatform/pelican/blob/469d87bf64a5f410f0dc751fcfcefec3609ba450/cmd/origin.go#L36) and it may be useful if we add any other components to the director like a web ui or monitoring.

Reply:
(in short, partially because I'm fond of the symmetry. If you think it adds unneeded cruft at this point, I'm not opposed to removing)

Reply:
That makes sense to me, although I think we put other components to `serveDirector` now. Let's keep this function and hopefully we put new components here in the future.